chain-evm/Cargo.toml

```diff
@@ -4,8 +4,9 @@ version = "0.1.0"
 edition = "2018"
 
 [dependencies]
+evm = "0.28.0"
 imhamt = { path = "../imhamt" }
-primitive-types = "0.10.1"
+primitive-types = "0.9.1"
```

Comment:
Why the version dowgrade?

Reply:
The reason for the version downgrade is that the evm crate uses "v0.9.1" as types returned in the traits needed to get the EVM runtime going. Specifying a different version in our manifest causes errors when compiling.

Reply:
I updated evm to 0.30.0 and primitive-types to 0.10.1